ft-fine-tune-bert/fine-tune-unittests.py

```diff
@@ -0,0 +1,118 @@
+import pandas as pd
+import unittest
+from finetune import DataTrainingArguments, get_dataset
+import os
+import pathlib as pl
+import transformers
+from transformers import PreTrainedTokenizer, LineByLineTextDataset, TextDataset, AutoTokenizer
+
+print(os.getcwd())
+
+class TestFT(unittest.TestCase):
+    def setUp(self):
+        """Overrides setUp from unittest to create a toy pandas dataframe for testing"""
+        # Here anomalies and non-anomalies are represented by -1 and 1 respectively.
+        self.train_data = [
```

Comment:
probably should dump the train_data into external file to be read from? So it could be swapped as needed. 

yaml or json comes to mind.

Reply:
Add two external files: unittests_eval_data.txt and unittests_training_data.txt for Unittests to read data from.